packages/svelte/src/lib/hooks/useSvelteFlow.ts

```diff
@@ -121,6 +127,10 @@ export function useSvelteFlow(): {
   return {
     zoomIn,
     zoomOut,
+    getNode: (id) => get(nodeLookup).get(id),
+    getNodes: (ids) => ids.map((id) => get(nodeLookup).get(id)),
```

Comment:
I would expect to be able to call `getNodes` without passing any ids to get all nodes here